src/bootstrap/setup.rs

```diff
@@ -50,6 +50,7 @@ impl fmt::Display for Profile {
 }
 
 pub fn setup(src_path: &Path, profile: Profile) {
+    const VERSION: usize = 2;
```

Comment:
Can you remove `VERSION` from `bin/main.rs` and re-export this? That way they'll be kept in sync and you won't have to update it in two places.